lib/moonshot/shell.rb

```diff
@@ -1,7 +1,17 @@
 require 'thor'
+require 'retriable'
 
 # Mixin providing the Thor::Shell methods and other shell execution helpers.
 module Moonshot::Shell
+  DEFAULT_RETRY_OPTIONS = {
+    on: RuntimeError,
+    tries: 50,
+    multiplier: 1,
+    base_interval: 1,
```

Comment:
I feel like the default base interval should be a bit longer.  Retrying an execution every second for 5 minutes doesn't seem like the best default setting.